src/pymovements/dataset/dataset_files.py

```diff
@@ -27,6 +27,7 @@
 import polars as pl
 from tqdm.auto import tqdm
 
+from pymovements._utils._paths import match_filepaths
```

Comment:
this seems to be direct merge conflict with #1029 where it was imported as 

`import match_filepaths as _match_filepaths`